static/css/style.css

```diff
@@ -5,7 +5,7 @@
 body {
     background-color: #f8f8f8;
     color: #3f3f3f;
-    font-family: 'Charter', Fallback, serif;
+    font-family: 'Charter', Helvetica, Arial, serif;
```

Comment:
This is loading a serif custom font (Charter) but the system fallbacks are sans-serif (Helvetica, Arial). That will look a little weird if (when?) the custom fonts don't load. Consider instead:

```
Charter, Times, "Times New Roman", serif
```

Reply:
Resovled by c8cd818